src/Network/Network/FirewallPolicy/PolicySettings/AzureApplicationGatewayFirewallPolicySetting.cs

```diff
@@ -74,6 +86,16 @@ public override void ExecuteCmdlet()
                 this.State = "Enabled";
             }
 
+            if (!this.MyInvocation.BoundParameters.ContainsKey("RequestBodyEnforcement"))
+            {
+                this.RequestBodyEnforcement = true;
```

Comment:
What will happen if a user is doing a Get where this field is not set and then configures this and Request Body Check is disabled. Shouldn't this be False?

Same comment for below variable.

@karanbazaz thoughts?